DEPATISnet.js

```diff
@@ -197,13 +197,14 @@ function scrape(doc, url) {
 		snapshot: false
 	});
 	
-	var pages = ZU.xpathText(doc, '//div[@id="inhalt"]/h2');
+	var pages = ZU.xpathText(doc, '/html/body/div[4]/p/span');			// old line: var pages = ZU.xpathText(doc, '//div[@id="inhalt"]/h2');
+	
 	// e.g. "Dokument   DE000004446098C2   (Seiten: 8)"
 	// but there is no PDF available when we have "Seiten: 0"
 	if (pages && /(Seiten|Pages):\s*[1-9][0-9]*/.test(pages)) {
 		var pdfurl = "https://depatisnet.dpma.de/DepatisNet/depatisnet/" + pn + "_all_pages.pdf?window=1&space=menu&content=download_doc_verify&action=download_doc&docid=" + pn;
 		newItem.attachments.push({
-			title: "Fulltext",
+			title: pn,		// old line:  title: "Fulltext",
```

Comment:
What's an example of what will be contained here, and what's the reason for the change? Generally "Full Text PDF" would be appropriate for the attachment item.

Reply:
I changed this (for me) because I wanted to have the patent number somewhere in the entry in the overview. I can see that it might make more sense to label the individual elements of the entry according to their category; e.g. 'full text' or 'comment', ... .
However, in order for the patent number to still appear with the entry, I suggest naming the entry with patent number and title, e.g.:

DE1234567A1 - Propulsion of a spacecraft by means of warp drive.

Would this be a feasible way?